site/content/blog/2020-06-04-zap-2-9-0-highlights/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "ZAP 2.9 Highlights"
+description: "Highlights from ZAP release 2.9.0"
+summary: "Do you know what interesting bits were added to ZAP 2.9.0? Don't read release notes? This blog post is for you! 
+Session Management Scripts, Proxy Info Display, Proxy Port Reservation Failure Handling, Options Panel(s) Filter, Active Scan Filter, and more."
+images:
+- https://www.zaproxy.org/blog/2020-06-04-zap-2-9-0-highlights/images/zapbot.png
+type: post
+tags:
+- blog
+- youdontknowzap
+date: "2020-06-04"
+addSocialPreview: true
+authors:
+    - thorin
+---
+
+Do you know what interesting bits were added to ZAP 2.9.0?
```

Comment:
I think its worth mentioning that 2.9.0 was released a while ago.

Reply:
And mention that there are other changes, but that this just goes into a few of the new features in more depth?